src/js/edu-benefits/1990e/config/form.js

```diff
@@ -126,6 +139,29 @@ const formConfig = {
     schoolSelection: {
       title: 'School Selection',
       pages: {
+        schoolSelection: {
+          title: 'School selection',
+          path: 'school-selection',
+          uiSchema: {
+            educationProgram: educationProgram.uiSchema,
+            educationObjective: {
+              'ui:title': 'Education or career goal (for example, “Get a bachelor’s degree in criminal justice” or “Get an HVAC technician certificate” or “Become a police officer.”)',
+              'ui:widget': 'textarea'
+            },
+            nonVaAssistance: {
+              'ui:title': 'Are you getting, or do you expect to get any money (including, but not limited to, federal tuition assistance) from the Armed Forces or public health services for any part of your coursework or training?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              educationProgram: educationProgram.schema(),
+              educationObjective,
+              nonVaAssistance
```

Comment:
Yeah, I was on the fence about that. But we are going to have at least three forms with this page, so maybe it's worth it.